global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "3.1.402"
```

Comment:
Since the SDK is currently pinned to 3.1.300 in init.yml (and according to @anpaz we can't update that version until QDK pipeline problems are sorted out), can we make that the minimum version here with a roll-forward?

Reply:
yes, makes sense